src/plotter_disk.hpp

```diff
@@ -364,6 +364,7 @@ class DiskPlotter {
 
         bool bCopied = false;
         bool bRenamed = false;
+        Timer copy;
```

Comment:
did you deliberately include possible wait time in this timer?

Reply:
Yes. The phase 1/2/3/4 and overall timers are all wall clock timers and include any delays due to "5 min. later" retries, process suspension, etc., so I figured this is a consistent approach to measuring the "copy phase" at the end.